### ISSUE TITLE
WebDav: Parsing response, using prefix: D:

### ISSUE THREAD

**Issue by lemorak** [J-D-K/JKSV#254]
Hi,

I tried to set up the webdav upload. First the log file said I have to create JKSV folder an there the game folder.

Now I have the following error in my log file.

WebDav: Response from WebDav. Parsing.
WebDav: Parsing response, using prefix: D:

Any ideas?

Greetings

**Comment by impeeza** [comment]
Have you reading https://github.com/J-D-K/JKSV/blob/master/REMOTE_INSTRUCTIONS.MD ?  the WevDav Support is on beta stage but works for some people,  JK is rewriting all the code and a robust support will come in the future.

**Comment by UnderPsi** [comment]
I had similar issues trying to use webdav on my synology NAS, but I read that it worked on windows IIS so I set that up and now it's working fine. At the moment I think not all webdav servers are supported.

**Comment by impeeza** [comment]
> I had similar issues trying to use webdav on my synology NAS, but I read that it worked on windows IIS so I set that up and now it's working fine. At the moment I think not all webdav servers are supported.

You are right the WEBDav support is on early stages JK is rewriting the code from scratch so far no so much correction will be done.

**Comment by swebb69** [comment]
Good to know about WebDAV in early stages and JK working towards something more robust! I've just spent the last 1h working through the IIS setup on my home server (Win Server 2k19) and trialing connectivity with JKSV to no avail, only to just now find your comments on this Issue thread! xD

**Comment by J-D-K** [comment]
I didn't write the original WebDav code. If this is still an issue with the rewrite's code, let me know and I'll reopen this issue. It shouldn't be though, because my WebDav code ignores the namespaces entirely. If I did write the original WebDav code, it would actually work like it does now.
